docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2858,6 +2861,29 @@ definitions:
         description: |
           Number of milliseconds that Nakadi stores events published to this event type.
 
+
+  EventAuthField:
+    type: object
+    description: |
+      Parameters to define per-event authorization. This field can be used with an authorization plugin
+      to differentiate which authorized consumers for an event type can receive an event published to
+      the event type. In case the field is null or not present in the event, it will be allowed to be read
+      by all authorized consumers.
+      Once defined, the definition for EventAuthField for an event-type cannot be updated or removed.
+    properties:
+      path:
+        type: string
+        description: |
+          Path to a string field in the event that contains the a classifier, which can be used by an
+          authorization plugin (together with definitions in the authorization section) to define who can read the
+          event. The field may not be present in the schema definition for the event type. In case the field is not
```

Comment:
This is not very clear for me. It says that `The field may not be present in the schema definition for the event type` but then I think it doesn't make it clear that in case an `event` does not contain the specified `path` or its value is `null` then this event is not filtered and all authorized consumer can read it.

Reply:
And by `authorized consumers` it might be worth explaining that we are referring to those in the `authorization` section, which is a per-event-type authorization, in contrast to this one which is `per-event` authorization.